frontend/src/editor/speaker_colors.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react';
+import { Editor } from 'slate';
+
+export const SpeakerColorsContext = React.createContext<Record<string, string>>({});
+
+function getColor(n: number) {
+  const hue = 230 + n * 137.5; // https://en.wikipedia.org/wiki/Golden_angle
+  return `hsl(${hue}, ${Math.min(70 + n * 10, 100)}%, 75%)`;
```

Comment:
What do gain by modulating the saturation? 
Maybe we want to just use a hardcoded color progression?

Reply:
> What do gain by modulating the saturation?
more diversity in the colors. I thought this way we can generate infinite colors that have kinda optimal distance. Maybe we want a more involved algorithm though. Something like starting with color hashes and then iteratively spreading them out according to perceived color distance?

Reply:
Hmm I am not convinced. Modulating the saturation is only doing something for the first 4 colors. 
I think we would want to use equal luminance colors?
I am still in favor of just a hardcoded list. I don't think having more than say 20 distinct speaker colors will actually help anything.